### DISCUSSION TITLE
How to copy files to a container?

### DISCUSSION THREAD

**Discussion by paulomorgado** [microsoft/aspire#6826] [Q&A] [answered]
When one is using a remote docker engine, one can't mount files.

Is there a way to copy files to the created container before the container is run, like [testcontainers does with `WithResourceMapping`](https://dotnet.testcontainers.org/api/create_docker_container/#copying-directories-or-files-to-the-container?)

**Comment by davidfowl** [comment]
I have no idea how this works 😄

**Reply by paulomorgado** [reply]
Testcontainers allows operations on the running container, like executing and copying.

I have situations where I need to work with remote container engines and mount files doesn't work.

**Reply by davidfowl** [reply, marked as answer]
https://github.com/dotnet/aspire/pull/8121
